frontend/src/libs/components/card/styles.module.scss

```diff
@@ -72,6 +79,25 @@
   word-break: break-word;
 }
 
+.icon-right {
+  position: relative;
+  top: -60px;
+  left: 120px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 30px;
+  padding: 7px 6px;
+  background-color: transparent;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+}
+
+.icon-right:hover {
```

Comment:
do something like scale on hover, without a background color

Reply:
![image](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/assets/92929504/4afb250a-f380-4941-9254-0a66b21e878d)

Reply:
I guess it also could be a box-shadow

Reply:
@Liza-Veis what do you think about how i should highlight button with hover effect?

Reply:
I understand about highligting, but not with the background color. You can see @Liza-Veis's opinion here [https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/312#issuecomment-1723297609](url)

Reply:
please fix this as Olena said, we need to make a color deeper, without scale or background color change